src/actions/personView.js

```diff
@@ -57,6 +57,10 @@ export function createPersonViewColumn(viewId, data) {
     return ({ dispatch, getState, z }) => {
         const orgId = getState().org.activeId;
 
+        if(data.title.length > 80) {
+          data.title = data.title.substring(0,77) + '...';
```

Comment:
Deviating indentation on this line.

Reply:
Ah yeah it's because I tried a new distribution of vim which had 2 character-indentation as default :P fixed.